snmp/datadog_checks/snmp/data/profiles/cisco.yaml

```diff
@@ -0,0 +1,12 @@
+# Profile for Cisco  devices
+
+extends:
+  - _base_cisco.yaml
+
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "cisco"
+
+sysobjectid: 1.3.6.1.4.1.9.1.*
```

Comment:
I think it would be nice to have a simple e2e for vendor generic profiles too. Example:

https://github.com/DataDog/integrations-core/pull/14693

Reply:
Agreed, though if we already have profiles today that cover these devices perhaps we should make this a separate story?